decidim-core/app/views/layouts/decidim/header/_menu_breadcrumb_mobile_tablet.html.erb

```diff
@@ -2,7 +2,7 @@
 <div class="menu-bar__breadcrumb-mobile__dropdown-trigger">
   <span>
     <% breadcrumb_items.last(2).each_with_index do |item, i| %>
-      <% item_label = decidim_escape_translated(item[:label]) %>
+      <% item_label = decidim_escape_translated(item[:label]).html_safe %>
```

Comment:
@rsdeus decidim_escape_translated, shouldn't always be html_safe? or add a param to the function. 
looks like to me this function should return a safe string?

Reply:
I see the function applied in numerous places: https://github.com/search?q=repo%3Adecidim%2Fdecidim+decidim_escape_translated&type=code&p=1

And I see no reference to html_safe... meaning the issue is potentially everywhere?